Showcase/picosheet.red

```diff
@@ -49,7 +49,7 @@ repeat y 9 [
 			]
 			actors: context [
 				on-create: on-unfocus: function [f e][
-					f/color: none
+					f/color: white
```

Comment:
What problem are you trying to fix there? The yellow color already disappears in cells on unfocusing.